_site/ui/css/styles.css

```diff
@@ -152,7 +152,7 @@ h1 {
     width: 100%;
     height: 400px;
 }
-.melville-rome #map-container {
+.melville-rome #map-container, .melville-london #map-container {
```

Comment:
Not sure how we'd handle styles if we wanted to abstract everything from the front end, but worth discussing.